main/reference/zoe-api/zoe.md

```diff
@@ -3,300 +3,333 @@
-executing contract. The `installation` is the unique identifier for
+Returns a **Promise** for the contract **installation** used by the
+**instance**. An **instance** is the unique identifier for the running,
+executing contract. The **installation** is the unique identifier for
 the underlying code. This method can be used as part of a process to
-inspect the underlying code for a running contract `instance`.
+inspect the underlying code for a running contract **instance**.
 
 ```js
-const installation = await E(zoe).getInstallationForInstance(instance);
+const installation = await E(Zoe).getInstallationForInstance(instance);
 ```
 
-## `E(zoe).startInstance(installation, issuerKeywordRecord, terms, privateArgs)`
-- `installation` `{ERef<Installation>}`
-- `issuerKeywordRecord` `{IssuerKeywordRecord}` - optional
-- `terms` `{Object}` - optional
-- `privateArgs` `{Object}` - optional
-- Returns: `{Promise<StartInstanceResult>}`
+## E(Zoe).startInstance(installation, issuerKeywordRecord?, terms?, privateArgs?)
+- **installation** **ERef&lt;Installation>**
+- **issuerKeywordRecord** **IssuerKeywordRecord** - Optional.
+- **terms** **Object** - Optional.
+- **privateArgs** **Object** - Optional.
+- Returns: **Promise&lt;StartInstanceResult>**
 
-Create an instance of the installed smart contract (specified by
-the `installation` argument). You must also specify the
-instance's `issuers` (as key-value pairs) and `terms`
-for the contract.
+Creates an instance of the installed smart contract specified by
+the *installation* argument.
 
-The `issuerKeywordRecord` is an optional record mapping string names (keywords)
-to issuers, such as `{ Asset: quatlooIssuer}`. Keywords must begin
+The **issuerKeywordRecord** is an optional record mapping string names (keywords)
+to issuers, such as **FirstCurrency: quatlooIssuer**. Keywords must begin
 with a capital letter and must be ASCII. Parties to the contract will
 use the keywords to index their proposal and their payments.
 
-The `terms` are values used by this contract instance, such as the
-number of bids an auction will wait for before closing. These values may
+The **terms** are values used by this contract instance, such as the
+number of bids an auction w ill wait for before closing. These values may
 be different for different instances of the same contract, but the contract
-defines what variables need their values passed in as `terms`.
+defines what variables need their values passed in as **terms**.
 
-`privateArgs` are optional. Pass an object record here with any values
+**privateArgs** are optional. Pass an object record here with any values
 that need to be made available to the contract code, but which should
 not be in the public terms. For example, to share minting authority
-among multiple contracts, pass in the following as `privateArgs`:
+among multiple contracts, pass in the following as **privateArgs**:
 
 ```js
 { externalMint: myExternalMint }
```

Comment:
This example would be better if it was more like
```js
const privateArgs = { externalMint: myExternalMint };
// setup other params
const resultP = await E(Zoe).startInstance(installation, issuerKeywords, terms, privateArgs);
```

Reply:
Your change is better than what was in the docs, but I really really don't like either example. So I've completely deleted the code snippet, as well as the "for example" text in the paragraph above.